rinad/src/ipcm/configuration.cc

```diff
@@ -395,6 +398,43 @@ void parse_ipc_to_create(const Json::Value          root,
         }
 }
 
+void check_conf_file_version(const Json::Value& root, const std::string& f)
+{
+	std::string last_version, f_version;
+	char answer = 'x';
+	ifstream file;
+	size_t pos;
+	stringstream ss;
+
+	f_version = root.get("configFileVersion", f_version).asString();
+
+	pos = f.rfind("/");
+	if (pos == std::string::npos) {
+		ss << ".";
+	} else {
+		ss << f.substr(0, pos);
+	}
+	ss << "/" << CONF_FILE_CUR_VERSION;
+
+        file.open(ss.str().c_str());
+        if (file.fail()) {
+                LOG_ERR("Failed to open last config version file");
+                return;
```

Comment:
Here we should also fail.
We get rid of this problem if we hardcode version in this source file.